.github/workflows/ci.yaml

```diff
@@ -84,6 +84,22 @@ jobs:
       # Setting RUSTDOCFLAGS because `cargo doc --check` isn't yet implemented (https://github.com/rust-lang/cargo/issues/10025).
       - run: RUSTDOCFLAGS="-D warnings" cargo doc --all --no-deps
 
+  scripts:
+    name: "test scripts"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: nightly-2022-11-01
```

Comment:
I think this can probably be changed _not_ to use nightly.

Reply:
Good call! Changed :)